app/src/main/java/org/stepik/android/domain/filter/model/CourseListFilterQuery.kt

```diff
@@ -0,0 +1,16 @@
+package org.stepik.android.domain.filter.model
+
+import android.os.Parcelable
+import com.google.gson.annotations.SerializedName
+import kotlinx.android.parcel.Parcelize
+import java.io.Serializable
+
+@Parcelize
+data class CourseListFilterQuery(
+    @SerializedName("language")
+    val language: String? = null,
+    @SerializedName("is_paid")
+    val isPaid: Boolean? = null,
+    @SerializedName("with_certificate")
+    val withCertificate: Boolean? = null
+) : Parcelable, Serializable
```

Comment:
let add here `toMap` too in order to reduce duplication in `SearchResultQuery`  & `CourseListQuery`